metricbeat/module/kubernetes/util/prometheus.go

```diff
@@ -0,0 +1,48 @@
+package util
+
+import (
+	"fmt"
+
+	"github.com/elastic/beats/metricbeat/helper"
+
+	dto "github.com/prometheus/client_model/go"
+	"github.com/prometheus/common/expfmt"
+)
+
+func GetFamilies(http *helper.HTTP) ([]*dto.MetricFamily, error) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function GetFamilies should have comment or be unexported